bucket-handlers.go

```diff
@@ -30,7 +30,7 @@ import (
 // http://docs.aws.amazon.com/AmazonS3/latest/dev/using-with-s3-actions.html
 func enforceBucketPolicy(api objectAPIHandlers, action string, bucket string, reqURL *url.URL) (s3Error APIErrorCode) {
```

Comment:
It would be uniform if all functions that work on bucket policy take ObjectLayer interface as an argument. i.e,
```
func enforceBucketPolicy(obj ObjectLayer, action string, bucket string, reqURL *url.URL) (s3Error APIErrorCode)
```